packages/builder/src/components/design/settings/controls/ButtonActionEditor/actions/DeleteRow.svelte

```diff
@@ -45,6 +45,9 @@
     <Checkbox text="Require confirmation" bind:value={parameters.confirm} />
 
     {#if parameters.confirm}
+      <Label small>Confirm title</Label>
```

Comment:
NAB: Because this setting appears when `Require confirmation` is checked, you could probably just call the label *Title* and the user will know what that is, imo.

Reply:
Agreed - these could just be `Title` and `Text`.

Reply:
Sweet, I've made that change, thanks guys.